addons/project_timesheet_holidays/models/account_analytic.py

```diff
@@ -15,3 +15,9 @@ class AccountAnalyticLine(models.Model):
     def _unlink_except_linked_leave(self):
         if any(line.holiday_id for line in self):
             raise UserError(_('You cannot delete timesheets linked to time off. Please, cancel the time off instead.'))
+
+    @api.returns('self', lambda value: value.id)
+    def copy(self, default=None):
+        default = default if default is not None else {}
+        default['holiday_id'] = False
+        return super().copy(default)
```

Comment:
Why not adding `copy=False` on the field definition?

Reply:
mmh I forgot the existence of this option in the field ^^